src/styles/styles.scss

```diff
@@ -0,0 +1,121 @@
+@import 'globals';
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  max-width: 800px;
+  margin: auto;
+  margin-top: 50px;
+  padding-top: 10px;
+  padding-left: 50px;
+  padding-right: 50px;
+  border-radius: 20px;
+  box-shadow: rgba(0, 0, 0, 0.16) 0px 10px 36px 0px;
+  @media only screen and (max-width: 520px) {
```

Comment:
Weren't we making the max-width a variable, so it can be changed dynamically?

Reply:
done